apps/daimo-mobile/src/view/screen/keyRotation/DeviceScreen.tsx

```diff
@@ -70,7 +70,7 @@ export function DeviceScreen({ route, navigation }: Props) {
     pendingOp: {
       type: "keyRotation",
       status: OpStatus.pending,
-      slot: device!.slot,
+      slot: device?.slot || 1024, // 1024 means "Deleted device", which is not a real slot
```

Comment:
could do || -1 or something, or use some constant if 1024 is actually a sentinel for deleted device

Reply:
done